webapp/views/wpt-results.js

```diff
@@ -255,6 +255,9 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
     <template is="dom-if" if="[[shouldDisplayToggle(canViewInteropScores, pathIsATestFile)]]">
```

Comment:
This will only display the button for test view in places where interop scores can be viewed. Is that intentional?

Reply:
For now yes. If this is something that is accepted by default, I can add an else template case that only shows the default and test buttons